### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewProblems_Keyboard: Focus moves out of the tree view and goes to the left pane to the Explorer button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51990]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.3

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Problems.
3. Press tab or shift+tab key to view the focus order.

Actual:
Focus moves to the left pane (Explorer) after tab key press.

Expected:
Focus should remain in the treeview itself, after tab key press focus should move to the output tab and then to all the controls in that line.
And In the Problems tab, all the opened editor files should be accessible through tab key.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the correct sequence of the controls.

MAS Reference:
MAS2.4.3- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Attachment for Reference: 
[A11y VSCode Problems NVDA Focus Order.pptx](https://github.com/Microsoft/vscode/files/2105416/A11y.VSCode.Problems.NVDA.Focus.Order.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
As far as I understand this is accessiblity inside the problems view, forawring to @sandy081

**Comment by sandy081** [comment]
@isidorn If I understand correctly, the issuer says that tabbing out of a panel is moving to explorer. Instead it has to navigate to `output` title. This is true for all panels.

**Comment by isidorn** [comment]
That is expected behavior. Another part of the issue says this
"And In the Problems tab, all the opened editor files should be accessible through tab key."

**Comment by sandy081** [comment]
`"And In the Problems tab, all the opened editor files should be accessible through tab key."` 

This is also expected as the tree and list navigation happens only with up and down arrows.
